{"name": "登录测试用例", "status": "broken", "statusDetails": {"message": "AttributeError: 'Android_UI_Client' object has no attribute 'driver'", "trace": "self = <cases.app_ui.android.test_login.TestLogin object at 0x0000023EC8225A10>, app = <base.android.android_ui_client.Android_UI_Client object at 0x0000023EC8262A50>\n\n    @pytest.fixture(scope=\"function\", autouse=True)\n    def setup(self, app):\n        self.ocr = OCRProcessor()\n>       self.lg = LoginPage(app.driver, app.appium_hub)\nE       AttributeError: 'Android_UI_Client' object has no attribute 'driver'\n\ncases\\app_ui\\android\\test_login.py:19: AttributeError"}, "description": "\n        登录测试用例,\n        ", "attachments": [{"name": "log", "source": "420f7746-46ee-4d41-9e23-bd1501ed140d-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'username': '834532523@qq.com', 'password': 'www123456', 'expected': '账号或密码错误', 'description': '密码错误'}"}], "start": 1757575922824, "stop": 1757575922824, "uuid": "f5d3b560-e5fd-4939-b104-6ffb5ed7839a", "historyId": "5aad62df486af26d243c29a74702a781", "testCaseId": "9b772e3c4d74e39c94b5c8b74ce1a0be", "fullName": "cases.app_ui.android.test_login.TestLogin#test_login", "labels": [{"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "cases.app_ui.android"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "wk"}, {"name": "thread", "value": "27072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.app_ui.android.test_login"}]}